src/NetAnalyzers/Core/Microsoft.CodeQuality.Analyzers/ApiDesignGuidelines/DoNotDirectlyAwaitATask.Fixer.cs

```diff
@@ -33,14 +33,14 @@ public sealed override async Task RegisterCodeFixesAsync(CodeFixContext context)
                 context.RegisterCodeFix(
                     new MyCodeAction(title,
                         async ct => await GetFix(context.Document, expression, argument: false, cancellationToken: ct).ConfigureAwait(false),
-                        equivalenceKey: title),
+                        equivalenceKey: nameof(MicrosoftCodeQualityAnalyzersResources.AppendConfigureAwaitFalse)),
```

Comment:
None of the changes in this file seem to be related to this PR. If so, can you please revert?

Reply:
This change gives a predicable equivalence key for the actions that we can use in testing. I might be able to submit it as a separate pull request, but the change is still needed.

Reply:
Sounds good, ideally we should keep these changes separate in future - fine to have them as-is in this PR. Thanks.